x/tokenfactory/keeper/createdenom_test.go

```diff
@@ -78,24 +95,55 @@ func (suite *KeeperTestSuite) TestCreateDenom() {
 			valid:    false,
 		},
 		{
-			desc:     "success case",
-			subdenom: "evmos",
-			valid:    true,
+			desc:             "success case: defaultDenomCreationFee",
+			denomCreationFee: defaultDenomCreationFee,
+			subdenom:         "evmos",
+			valid:            true,
 		},
 		{
-			desc:     "subdenom having invalid characters",
-			subdenom: "bit/***///&&&/coin",
-			valid:    false,
+			desc:             "success case: twoDenomCreationFee",
+			denomCreationFee: twoDenomCreationFee,
+			subdenom:         "catcoin",
+			valid:            true,
+		},
+		{
+			desc:             "success case: nilCreationFee",
+			denomCreationFee: nilCreationFee,
+			subdenom:         "czcoin",
+			valid:            true,
+		},
+		{
+			desc:             "account doesn't have enough to pay for denom creation fee",
+			denomCreationFee: largeCreationFee,
+			subdenom:         "tooexpensive",
+			valid:            false,
+		},
+		{
+			desc:             "subdenom having invalid characters",
+			denomCreationFee: defaultDenomCreationFee,
+			subdenom:         "bit/***///&&&/coin",
+			valid:            false,
 		},
 	} {
+		suite.SetupTest()
 		suite.Run(fmt.Sprintf("Case %s", tc.desc), func() {
 			if tc.setup != nil {
 				tc.setup()
 			}
-			// Create a denom
+			tokenFactoryKeeper := suite.App.TokenFactoryKeeper
+			bankKeeper := suite.App.BankKeeper
```

Comment:
I think this is getting excessive with aliasing. Don't need to revert or anything, but I'd suggest not feeling a strong need to interlace this pattern everywhere imo